src/Frontend/Page/Idea.hs

```diff
@@ -149,14 +149,13 @@ instance ToHtml ViewIdea where
                         h2_ [class_ "comments-header-heading"] $ do
                             totalComments ^. showed . html <> " Verbesserungsvorschläge"
                                 -- FIXME: code redundancy!  search for 'totalComments' in this module
+                        button_ [value_ "create_comment comments-header-button", class_ "btn-cta comments-header-button"] "Neuer Verbesserungsvorschlag"
```

Comment:
@local-girl I think there is a bogus class name here.  will fix it 239f6e8f in a moment.; could you please check if i got it right?